common/model/opening-hours.js

```diff
@@ -10,22 +10,37 @@ export type OpeningHoursDay = {|
   note?: string
 |};
 
-type ExceptionalOpeningHoursDay = {|
+export type ExceptionalOpeningHoursDay = {|
   ...OpeningHoursDay,
-  overrideDate: Date
+  overrideDate: Date,
+|}
+
+export type SpecialOpeningHours = {|
```

Comment:
Should we leave a not here explaining that this is for JSON+LD - or perhaps just put the type in there?